.github/workflows/sbom.yml

```diff
@@ -52,6 +52,7 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: '24'
+      - run: npm install -g npm@11.1
```

Comment:
Is this really needed here?

Reply:
this is only for the sbom checking
and with the npm 11.2+, there is this issue. https://github.com/npm/cli/issues/8688